adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Adaptadog</title>
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Francois+One%7CJacques+Francois" rel="stylesheet">
+</head>
+<body>
+    <!-- <header> -->
+    <header class="header">
+      <div class="container">
+        <!-- <img src="images/christopher-harris-57366-unsplash.jpg" alt="header image" height="400"> -->
+      <img src="adoptadog-logo.svg" alt="Adoptadog Logo" height="100">
+      <h2>Seattle's Premier Dog Adoption Agency</h2>
+    </div>
+      <div class="header-cart">
+        <img src="images/cart-icon.svg" alt="shopping cart icon" height="40">
+        <button class="button">Check Out</button>
+      </div>
+    </header>
+
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">/Dogs</a></li>
+        <li><a href="blog.html">/Blog</a></li>
+      </ul>
+    </nav>
+  <h2>Here are all of our Dogs!</h2>
+  <!-- Murphy -->
+  <div class="dog-tile-container">
+    <div class="tile">
+      <img class="tile img" src="images/alex-lacayo-756508-unsplash.jpg" alt="murphy profile">
```

Comment:
this image is taking two classes. a class of tile and a class of img. Image is going to be getting the same css that you apply to the wrapping container (and causing your dog photos to come out of the box